google-cloud-tasks/lib/google/cloud/tasks/v2beta2/doc/overview.rb

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Google
+  module Cloud
+    # rubocop:disable LineLength
+
+    ##
+    # # Ruby Client for Cloud Tasks API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+    #
+    # [Cloud Tasks API][Product Documentation]:
+    # Manages the execution of large numbers of distributed requests. Cloud Tasks
+    # is in Alpha.
+    # - [Product Documentation][]
+    #
+    # ## Quick Start
+    # In order to use this library, you first need to go through the following
+    # steps:
+    #
+    # 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+    # 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+    # 3. [Enable the Cloud Tasks API.](https://console.cloud.google.com/apis/api/cloudtasks)
+    # 4. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+    #
+    # ### Installation
+    # ```
+    # $ gem install google-cloud-tasks
+    # ```
+    #
+    # ### Next Steps
+    # - Read the [Cloud Tasks API Product documentation][Product Documentation]
+    #   to learn more about the product and see How-to Guides.
+    # - View this [repository's main README](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/README.md)
+    #   to see the full list of Cloud APIs that we cover.
+    #
+    # [Product Documentation]: https://cloud.google.com/cloudtasks
+    #
+    #
+    module Tasks
```

Comment:
@jbolinger I believe this is another instance of the "blank pages" issue in #2072. Is there some docs content available for the `V2beta2` module? I have confirmed that the jsondoc output for `google/cloud/tasks/v2beta2` will result in a blank page, since in that file: `"description":""`.

Reply:
Looks like the same issue to me.

@landrito - it looks like you generated Dataproc and it does include the doc comments that are missing here. For example, I think this is what we want (@quartzmo does that look right to you?): 

https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/e52957dd157b0d25d6badb3b5a3a244ff69e7e6f/google-cloud-dataproc/lib/google/cloud/dataproc/v1/doc/google/cloud/dataproc/v1/clusters.rb#L18-L31)

Why are we not seeing the same thing here? Was that manually added or was it changed in the generator?

Reply:
@jbolinger I think a table like that is the right content, but tbh I'm not 100% sure.